lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -2,6 +2,14 @@ module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmCloud
     expose :cloud_networks, :association => true
 
+    expose :resize,         :override_return => nil
+    expose :confirm_resize, :override_return => nil
+    expose :revert_resize,  :override_return => nil
```

Comment:
@aufi The methods on the app/model side are resize_*.   [providers/cloud_manager/vm.rb#L49](https://github.com/ManageIQ/manageiq/blob/33d0fd7d4b5093599825b628728bc7bdde15c81e/app/models/manageiq/providers/cloud_manager/vm.rb#L49)

The last two still need to be renamed.  `resize_confirm` and `resize_revert`